content/en/continuous_integration/setup_pipelines/jenkins.md

```diff
@@ -164,6 +164,27 @@ You can configure the Jenkins Plugin to send custom tags in all pipeline traces:
 **Environment variable**: `DATADOG_JENKINS_PLUGIN_GLOBAL_JOB_TAGS`<br/>
 **Example**: `(.*?)_job_(.*?)_release, owner:$1, release_env:$2, optional:Tag3`
 
+### Including/Excluding pipelines
+
+You can configure the Jenkins Plugin to include/exclude some pipelines:
+
+1. In the web interface of your Jenkins instance, go to **Manage Jenkins > Configure System**.
+2. Go to the `Datadog Plugin` section, scrolling down the configuration screen.
+3. Click on the `Advanced` button.
+4. Configure the `Excluded Jobs`.
+5. Configure the `Included Jobs`.
+6. Save your configuration.
+
+**Excluded jobs**
```

Comment:
```suggestion
#### Excluded jobs
```

Reply:
About this, I'm using the same approach used in the previous sections. Should we use a different approach here?

Reply:
ok, we can skip this for now. Headers just make it easier to navigate the page.